Node.js/README.md

```diff
@@ -0,0 +1,7 @@
+# Azure Batch Node.js sample
+The set of scripts provided in the [GettingStarted](##add link here##) folder is a sample of creating Azure Batch jobs using the Node.js SDK.
```

Comment:
It is better in readme to mention that user needs to fill in some missing account/key stuff in order to run the test, .